app/views/customers/_basic_fields.html.slim

```diff
@@ -0,0 +1,12 @@
+.form-group
+  label.control-label.col-md-3 for="email" 
+    | Email
+    span.required *
+  .col-md-6
+    = f.email_field :email, autofocus: true, class: 'form-control'
+.form-group
+  label.control-label.col-md-3 for="primary_phone_no" 
+    | Primary Phone No
+    span.required *
+  .col-md-6 
+    = f.text_field :primary_phone_no, class: 'form-control'
```

Comment:
@RaviMariya-BTC You have required mark on the `primary_phone_no` But where is the backend validation for the same.